QKSMS/src/main/res/values-sv/strings.xml

```diff
@@ -1,454 +1,454 @@
-  <string name="pref_navigation_compat">Navigation bar fix</string>
-  <string name="pref_navigation_compat_summary">Fixes issues caused by immersive mode, Tinted Status Bar (Xposed), or custom DPI settings</string>
-  <string name="pref_about_version_title">QKSMS version</string>
-  <string name="pref_about_authors_title">Authors</string>
+  <string name="pref_mms_proxy">MMS-proxy</string>
+  <string name="pref_mms_port">MMS-port</string>
+  <string name="pref_apn_chooser">Ladda standard-APN:er</string>
+  <string name="pref_strip_unicode">Strippa Unicode</string>
+  <string name="pref_strip_unicode_summary">Byt ut tecken som inte stöds i fillängdsmeddelanden för GSM-kompatibla tecken till kompatibla tecken</string>
+  <string name="pref_system_bar_flat">Platt systemfält</string>
+  <string name="pref_system_bar_flat_summary">Få bort stigningen som visas i systemgränssnittsfälten på de flesta KitKat-enheterna</string>
+  <string name="pref_status_compat">Fix för statusfält</string>
+  <string name="pref_status_compat_summary">Fixar problem som orsakas av platt systemfält (Xposed) eller anpassade DPI-inställningar</string>
+  <string name="pref_navigation_compat">Fix för navigeringsfält</string>
+  <string name="pref_navigation_compat_summary">Fixar problem som orsakas av platt systemfält (Xposed) eller anpassade DIP-inställningar</string>
+  <string name="pref_about_version_title">QKSMS-version</string>
+  <string name="pref_about_authors_title">Upphovsmän</string>
   <string name="pref_about_authors_summary">Moez Bhatti</string>
-  <string name="pref_changelog_summary">Tap to view changelog</string>
-  <string name="pref_about_thanks_title">Thank you</string>
-  <string name="pref_about_thanks_summary">Tap to view open-source contributors</string>
-  <string name="pref_donate_title">Donate</string>
-  <string name="pref_donate_summary">QKSMS is free. Help support us by donating!</string>
-  <string name="pref_show_all">Show all preferences</string>
-  <string name="pref_show_fewer">Show fewer preferences</string>
-  <string name="pref_subcat_theme">Theme</string>
-  <string name="pref_subcat_system_bars">System bars</string>
-  <string name="pref_subcat_categories">Categories</string>
-  <string name="pref_subcat_font">Font</string>
-  <string name="pref_subcat_messages">Messages</string>
-  <string name="pref_subcat_avatars">Avatars</string>
-  <string name="donate_google">Donate with Google</string>
-  <string name="donate_google_summary">Google takes 30% of donations</string>
-  <string name="donate_paypal">Donate with PayPal</string>
-  <string name="donate_paypal_button">Donate any amount</string>
-  <string name="donate_paypal_button_summary">Buy me anything you want!</string>
+  <string name="pref_changelog_summary">Tryck för att visa vad som är nytt</string>
+  <string name="pref_about_thanks_title">Tack/string>
```

Comment:
Oops, I'm so sorry.

@moezbhatti I can't upload the file. Could you please upload the file at Crowdin?